scripts/registration.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+//calling to the base API
+const apiBaseURL = "https://microbloglite.herokuapp.com";
+
+//new user request
+function newUser() {
+    const un = document.getElementById('UserName').value;
+    const fn = document.getElementById('fullName').value;
+    const ps = document.getElementById('password').value;
+
+    const requestBody = {
+        username: un,
+        fullName: fn,
+        password: ps
+    };
+
+    //fetching from api
+    fetch(apiBaseURL + '/api/users', {
+        method: 'POST',
+        headers: {
+            "Content-Type": "application/json",
+        },
+        body: JSON.stringify(requestBody)
+    })
+        .then(response => response.json())
+        .then(window.location.href='index.html')
```

Comment:
I'd make this a function object/definition rather than a statement, so something like 
```
.then(() => window.location.href = 'index.html')
```